cmd/katib-controller/v1alpha1/Dockerfile

```diff
@@ -7,10 +7,8 @@ ADD . /go/src/github.com/kubeflow/katib
 WORKDIR /go/src/github.com/kubeflow/katib/cmd/katib-controller
 # Build
 RUN CGO_ENABLED=0 GOOS=linux GOARCH=amd64 go build -a -o studyjobcontroller  ./v1alpha1
-RUN CGO_ENABLED=0 GOOS=linux GOARCH=amd64 go build -a -o katib-controller.v1alpha2  ./v1alpha2
 # Copy the controller-manager into a thin image
 FROM alpine:3.7
 WORKDIR /app
 COPY --from=build-env /go/src/github.com/kubeflow/katib/cmd/katib-controller/studyjobcontroller .
-COPY --from=build-env /go/src/github.com/kubeflow/katib/cmd/katib-controller/katib-controller.v1alpha2 .
```

Comment:
If we want to support both v1alpha1 and v1alpha2 with same image, don't we need to build both of the versions together?
@richardsliu

Reply:
v1alpha1 and v1alpha2 have big difference, I think maybe we should separate them in two images.

Reply:
But this is the docker file for v1alpha1, based on the file path. Why is it building v1alpha2?

Reply:
For tf-operator and pytorch-operator, we've always built multiple versions with the same image. In the pod template we override the entrypoint for different versions.

Reply:
OK, I can follow operators.